doc/modules/mod_muc_light_commands.md

```diff
@@ -0,0 +1,49 @@
+# MongooseIM's multi-user chat light commands set
+
+## Purpose
+This is a set of commands, providing actions connected with multi-user chat light features.
+It is currently used to provide commands used for exposing REST API for managing rooms.
```

Comment:
"It is currently used to provide commands used to expose REST API methods designed for managing rooms."?